CHANGELOG.md

```diff
@@ -1,5 +1,86 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.23
+
+###### _Jan 14, 2021_
+
+Big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 📚 Support public paths in module augmentation (#24267) @eps1lon
+- 👩‍🎤 Migrate the Paper to emotion (#24397) @povilass
+- 📚 Implemented various CI improvements (#24313) @eps1lon, (#24358) @oliviertassinari
+
```

Comment:
We did more work on it than usual, maybe we can mention it. 

```suggestion
- 📅 Polish the date picker components  
```

Reply:
We should make it clear that these only changed internals i.e. "Refactor internals of the picker components". 

Changelogs are about what library consumers need to look out for. "Polish" is too ambigious in my opinion.

Reply:
We have done more than improving the internals

Reply:
Updated to 

> - 📅 Various impromenets on the picker components (#24301, #24309, #24275, #24298, #24319) @m4theushw @eps1lon @huzaima @praveenkumar-kalidass

Reply:
No, we shouldn't mix actual bugs with internal refactorings. This might make sense for a blog post but not a changelog. A changelog is a strictly technical document.

Each bug fix/feature needs its separate changelog entry.

The internal refactorings can be grouped as a slight warning: "keep an eye out for changes in behavior for these components. We did some internal changes that are not intended to change documented behavior".

Reply:
Internal: #24309, #24275, #24319
Public: #24301, #24298

Reply:
Should we then even mention these as highlights? Or maybe just the public ones?